packages/main/src/RPA/Excel/Files.py

```diff
@@ -582,7 +613,13 @@ def read_worksheet(self, name=None, header=False, start=None):
         assert self.workbook, "No active workbook"
         return self.workbook.read_worksheet(name, header, start)
 
-    def read_worksheet_as_table(self, name=None, header=False, trim=True, start=None):
+    def read_worksheet_as_table(
+        self,
+        name: Optional[str] = None,
+        header: Optional[bool] = False,
+        trim: Optional[bool] = True,
```

Comment:
```suggestion
        header: bool = False,
        trim: bool = True,
```
It's not commonly expected to pass `None`s to booleans like these. Usually you rely on its default value (by not specifying the param during the call) or send `True`/`False` explicitly, therefore no `Optional` annotation.